src/lib/config/defaultConfig.js

```diff
@@ -117,7 +117,7 @@ export const RECORDS_CONFIG = {
       sort: [
         {
           order: 1,
-          sortBy: 'mostrecent',
+          sortBy: '-mostrecent',
```

Comment:
I would have opted for something more understandable:
```
sortBy: 'mostrecent',
sortOrder: 'desc',
```
but I am fine with this :)

Reply:
Without the minus in front the order is the opposite (less recent)

Reply:
I am quickly checking this point in my local instance

Reply:
As discussed, please remove this change, I will create 2 PRs to fix frontend/backend